packages/angular_devkit/build_angular/src/webpack/configs/common.ts

```diff
@@ -101,6 +105,19 @@ export async function getCommonConfig(wco: WebpackConfigOptions): Promise<Config
   // determine hashing format
   const hashFormat = getOutputHashFormat(buildOptions.outputHashing);
 
+  if (!vendorSourceMap) {
```

Comment:
Can you actually put this under https://github.com/angular/angular-cli/blob/c4401882d6e46ed79a2190501500dd06000d2182/packages/angular_devkit/build_angular/src/webpack/configs/common.ts#L200 so that this plugin is only enabled when sourcemaps are enabled?

Reply:
Done.